microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -247,12 +252,16 @@ def generate_testpmd_command(
         #   --eth-peer=<port id>,<receiver peer MAC address> \
         #   --stats-period <display interval in seconds>
 
+        # txq and rxq will always be equal in our testing
         if multiple_queues:
-            txq = 4
-            rxq = 4
+            if self.is_mana:
+                queues = 8
```

Comment:
Use the previous name `rxq` may be clearer.

Reply:
I think queues is better as long as the value is shared for both the txq and rxq flags. I'll switch it back if we move to support some other setup.